manual/C/ch_Finance-Quote.xml

```diff
@@ -207,10 +214,14 @@ SYNOPSIS
       </step>
 
       <step>
-        <simpara>Install Xcode if it is not installed.
-        </simpara>
-
-        <simpara>XCode is an optional item from your &mac; distribution DVD.
+        <simpara>Install &app-xcode;
+          <footnote>
+            <para>&app-xcode; is an optional tool from your
+              &mac; distribution DVD, but also at <ulink url="&url-ap-dev;xcode/" />
+              the application is also available.
+            </para>
```

Comment:
There hasn't been a macOS distribution DVD in a very long time, maybe since the versions were named after cats. Nowadays you download Xcode from the App Store. But there's a much smaller install that will work as well, named Xcode command line tools. Let's make this:
```
<para>You can install &app-xcode; from the App Store or install the much smaller Xcode command line tools by running <tt>sudo xcode-select --install</tt> from a Terminal.app prompt.
</para>
```